musicHelper.md

```diff
@@ -0,0 +1,128 @@
+#Piano Music Helper
+
+###_Introduction_
+This bot helps you find a piano note of the famous word composers 
+
+at pdf or a doc documents for printing it for your device
+
+
+##User story for people who play with piano
+As a telegram bot user
+
+I want to find a note with good type of docs
+
+So that I can  print it or download on my devise
+
+And play at my piano
+
+###_start of the screen scenario_
+Given a user selection of genre he may choose 
+
+Then the telegram bot will display a list of genres 
+
+###_Scenario of the selected genre_
+Given a user opportunity to finish the process of choose
+
+Then he just stops the work of telegram bot
+
+And will return one step back
+
+###_Scenario of save the piano notes_
+Given a user possibility to save notes for his device
```

Comment:
I do not understand this scenario, as well as the precondition for it. I think that we might need to discuss it offline